website/admin/views/admin_user_rooms.py

```diff
@@ -0,0 +1,51 @@
+from flask import redirect, url_for, flash, render_template
+
+import data_interface
+from admin import admin_site
+from admin.views.admin_user_forms import AddNewRoomForm
+
+
+@admin_site.route('/user/<string:user_id>')
+def user(user_id):
+    rooms = data_interface.get_default_rooms_for_user(user_id)
```

Comment:
This is a good example of how to pass the user id as a parameter. You could use this for the other endpoints as well.